### DISCUSSION TITLE
Exporting installed models back to   .tar.gz files.

### DISCUSSION THREAD

**Discussion by foscraft** [explosion/spaCy#10126] [Help: Coding & Implementations] [answered]
I am working on my spacy project, and I want to make it easy for users to add entity ruler to the models using the yaml file easily. As part of this I want to find a way to export an already installed model back to tar.gz file.

How can I reverse that?

**Comment by polm** [comment, marked as answer]
I don't think we'd considered that use case before, but you should be able to do it by using `spacy package` with the directory where the model was installed. You can find the directory with `pip show [model_name]`.

**Reply by foscraft** [reply]
1. Am getting this error.

` spacy package /home/miniconda3/envs/spacy3.1.0/lib/python3.8/site-packages/ .`
ℹ Building package artifacts: sdist

✘ Can't load pipeline meta.json
`/home/miniconda3/envs/spacy3.1.0/lib/python3.8/site-packages/meta.json`

2. Also after running this.
`spacy package /home/foscraft/miniconda3/envs/spacy3.1.0/lib/python3.8/site-packages/en_core_web_sm/ .`

I get this error:
`OSError: [E053] Could not read config.cfg from /home/miniconda3/envs/spacy3.1.0/lib/python3.8/site-packages/en_core_web_sm/config.cfg`

**Reply by polm** [reply]
Sorry for not being clearer you need to use the directory with the `config.cfg` file, something like `site-packages/en_core_web_sm/en_core_web_sm-3.2.0/` (so one deeper than you are using now).

**Reply by foscraft** [reply]
Thank you!. It worked.
